certified-connectors/DocuSignDemo/script.csx

```diff
@@ -1726,6 +1726,34 @@ public class Script : ScriptBase
       response.Content = new StringContent(newBody.ToString(), Encoding.UTF8, "application/json");
     }
 
+    if ("GetTemplateDocumentTabs".Equals(this.Context.OperationId, StringComparison.OrdinalIgnoreCase))
+    {
+      var body = ParseContentAsJObject(await response.Content.ReadAsStringAsync().ConfigureAwait(false), false);
+      var newBody = new JObject();
+      JArray tabs = new JArray();
+
+      foreach(JProperty tabTypes in body.Properties())
+      {
+        foreach(var tab in tabTypes.Value)
+        {
+          tabs.Add(new JObject()
```

Comment:
This is ok to merge now.
But in the next PR, can you consider revisiting refactoring the code from line 1735-1750 into a separate method. I believe we duplicated that for 3 actions. GetRecipientTabs, GetEnvelopeDocumentTabs, and GetTemplateDocumentTabs. If all 3 are similar then we can group it.